pkgs/development/compilers/ghc/common-hadrian.nix

```diff
@@ -638,15 +636,12 @@ stdenv.mkDerivation (
 
     nativeBuildInputs =
       [
+        autoreconfHook
```

Comment:
Do you know whether this will overwrite `config.sub` / respects `dontUpdateAutotoolsGnuConfigScripts`? Otherwise this is a problem since GHC ships a non standard config.sub which is important for ghcjs.

Reply:
I was able to build a ghcjs cross compiler, so it seems to be alright.